docs/snap-clone.md

```diff
@@ -159,41 +160,50 @@ kubectl create -f snapshot.yaml
 
 ```console
 $ kubectl get volumesnapshot
-NAME               AGE
-rbd-pvc-snapshot   6s
+NAME               READYTOUSE   SOURCEPVC   SOURCESNAPSHOTCONTENT   RESTORESIZE   SNAPSHOTCLASS                            SNAPSHOTCONTENT                                    CREATIONTIME   AGE
+rbd-pvc-snapshot   true         rbd-pvc                             1Gi           csi-rbdplugin-snapclass                  snapcontent-905e6015-2403-4302-8a4e-cd3bdf63507b   78s            79s
 ```
 
 ### Check the status of the Snapshot
 
 ```console
 $ kubectl describe volumesnapshot rbd-pvc-snapshot
```

Comment:
sorry to back here again for me looks like both cephfs and rbd snapshot/clone steps are different, we don't do describe for cephfs snapshots

Reply:
@Madhu-1 updated this part too. ptal.. thanks

Reply:
lets not have describe output for both? IMO describe output is not useful

Reply:
@Madhu-1  Well I added describe output  by following the comment above ```steps are different..we dont do describe for cephfs snapshots...` , now is the ask to remove it ?

Reply:
Also having `describe` should be fine too..

Reply:
>@Madhu-1 Well I added describe output by following the comment above ```steps are different..we dont do describe for cephfs snapshots...` , now is the ask to remove it ?

@humblec I never asked to add it to cephfs :)

Reply:
@Madhu-1 so remove the describe from both  ? already gone through different iterations here, so please clarify. IMO, having describe output is fine and its consistent across RBD and CephFS here.

Reply:
IMO having described output is not useful. my suggestion is to remove it

Reply:
@Madhu-1 done, ptal.